feathr_project/feathr/registry/_feathr_registry_client.py

```diff
@@ -196,185 +196,6 @@ def _post(self, path: str, body: dict) -> dict:
     def _get_auth_header(self) -> dict:
         return {"Authorization": f'Bearer {self.credential.get_token("https://management.azure.com/.default").token}'}
```

Comment:
@xiaoyongzhu I believe there is another azure coupling here as well

Reply:
Good point, let me update

Reply:
@aabbasi-hbo I was looking at it and feel it might make sense to have a follow up PR. Basically I think the current auth is coupled with Azure credentials, so in that PR I think I'll separate the auth part.

I have opened this github issue to track this:
https://github.com/feathr-ai/feathr/issues/873